src/components/TimePicker/styled/selectValue.js

```diff
@@ -18,64 +27,64 @@ const StyledSelectValue = styled.input`
     line-height: normal;
 
     &::selection {
-        color: ${COLOR_WHITE};
-        background: ${COLOR_BRAND_ACTIVE};
+        color: ${props => props.getContrastText(props.brandDarkColor)};
+        background: ${props => props.brandDarkColor};
     }
 
     &::-moz-selection {
-        color: ${COLOR_WHITE};
-        background: ${COLOR_BRAND_ACTIVE};
+        color: ${props => props.getContrastText(props.brandDarkColor)};
+        background: ${props => props.brandDarkColor};
     }
 
     &[placeholder]:focus::-webkit-input-placeholder {
-        color: ${COLOR_WHITE};
+        color: ${props => props.getContrastText(props.brandDarkColor)};
     }
 
     &[placeholder]:focus::-moz-placeholder {
-        color: ${COLOR_WHITE};
+        color: ${props => props.getContrastText(props.brandDarkColor)};
     }
 
     &[placeholder]:focus:-ms-input-placeholder {
-        color: ${COLOR_WHITE};
+        color: ${props => props.getContrastText(props.brandDarkColor)};
     }
 
     &[placeholder]:focus:-moz-placeholder {
-        color: ${COLOR_WHITE};
+        color: ${props => props.getContrastText(props.brandDarkColor)};
     }
 
     &::-webkit-input-placeholder {
-        color: ${COLOR_BRAND};
+        color: ${props => props.brandMainColor};
         font-size: 40px;
         font-weight: 200;
         text-transform: uppercase;
     }
 
     &::-moz-placeholder {
-        color: ${COLOR_BRAND};
+        color: ${props => props.brandMainColor};
         font-size: 40px;
         font-weight: 200;
         text-transform: uppercase;
     }
 
     &:-ms-input-placeholder {
-        color: ${COLOR_BRAND};
+        color: ${props => props.brandMainColor};
         font-size: 40px;
         font-weight: 200;
         text-transform: uppercase;
     }
 
     &:-moz-placeholder {
-        color: ${COLOR_BRAND};
+        color: ${props => props.brandMainColor};
         font-size: 40px;
         font-weight: 200;
         text-transform: uppercase;
     }
 
     &:focus,
     &:active {
-        color: ${COLOR_WHITE};
-        background-color: ${COLOR_BRAND};
-        box-shadow: ${SHADOW_OUTLINE};
+        color: ${props => props.getContrastText(props.brandMainColor)};
+        background-color: ${props => props.brandMainColor};
+        box-shadow: 0 0 2px ${props => props.brandMainColor};
```

Comment:
Pass the brandShadow through attrs function like we did in src/components/Button/styled/button.js.
Also add the TODO comment added in Button in order to know where refactor when we define the shadows.